docs/tokenomics/token-holders/nns-app-quickstart.md

```diff
@@ -264,6 +264,50 @@ To disburse a neuron and return its ICP utility tokens:
 
 - #### Step 4:  Verify the transaction information, then click **Confirm and Send**. 
 
+## Neuron following
+
+To maximize neuron rewards and fully utilize the NNS's liquid democracy, neuron owners are encouraged to vote on every proposal submitted to the NNS. However, it may be hard for neuron owners to manually vote on every single proposal submitted, since several proposals may be submitted at one time, or neuron owners may not have the necessary level of technical understanding to evaluate proposals themselves. To aid in this, the NNS supports **neuron following**.
+
+A neuron can be configured to vote automatically by following the voting decision made by a group of other neurons. For example, if the neuron follows a group that votes to adopt a proposal with more than half of the followees voting in favorite of adoption, the neuron following that group will vote to approve the proposal as well. If the neuron follows a group that votes to reject a proposal with more than half of the followees voting to reject, the neuron following the group will vote to reject the proposal. 
+
+Additionally, there is a catch-all follow rule that may be defined for a neuron to automatically vote on proposals for which no follow rule has been defined. It is assumed that neuron owners will follow other neurons that support the best interests of the network and their own economic interests.
+
+To configure your neuron to follow the group of another neuron:
+
+- #### Step 1: From the NNS dapp, select 'My Neuron Staking', then select the neuron that you'd like to set a following rule for.
+
+![NNS Neuron following 1](../_attachments/following1.png)
+
+- #### Step 2: In the neuron's details, scroll down to the 'Following' section and select 'Follow Neurons'. 
+
+![NNS Neuron following 2](../_attachments/following2.png)
+
+- #### Step 3: Next, select a topic to follow. 
+
+The topic refers to the different types of proposals; for example, the 'Governance' topic defines a following role for governance type proposals, while the 'Subnet Management' topic defines a following role for subnet management proposals. There is also a 'catch-all' rule that can be followed for all proposal types except for governance, SNS, and neurons' fund. 
+
+For this example, select 'Governance'. 
+
+![NNS Neuron following 3](../_attachments/following3.png)
+
+- #### Step 4: Select 'Add Followee'.
+
+![NNS Neuron following 4](../_attachments/following4.png)
+
+- #### Step 5: Enter a Neuron ID to follow, or select a group of neurons from the menu provided. 
+
+In this example, select 'DFINITY Foundation'. 
```

Comment:
We might want to consider making another example here to encourage people to try out something else and note how to get someon else's neuron ID . But this can also be an improvment for later.